log.toml

```diff
@@ -0,0 +1,8 @@
+[appenders.async_file]
```

Comment:
Would rather put this file in `sample-log-file` dir instead. Being present here makes it seem like it's important to the project in more ways than just a serving sample.